### PR TITLE
Added some docstrings ops/nn.py

### PR DESCRIPTION
**Pull request by Frightera**
I added some missing docstrings to the activation functions.

### CODE REVIEW

File: keras_core/ops/nn.py [hunk 1]
```diff
@@ -22,6 +22,24 @@ def compute_output_spec(self, x):
...
+
+    Args:
+        x: Input tensor.
+
+    Returns:
+        A tensor with the same shape as `x`.
+
+    Examples:
+
+        >>> x = np.array([-1, 0, 1])
```
Comment (fchollet) [inline keras_core/ops/nn.py:37]: 1. Unindent (the `>>>` should be at docstring level
2. Prefer using floats since that's the typical activation use case

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

**Comment by Frightera** [comment]
Thanks for the review @fchollet.

Is there any reason both scale and alpha parameters are omitted in selu activation here?

https://github.com/keras-team/keras-core/blob/db9d7342815e6a98e3a209718f7de135b3a554fa/keras_core/ops/nn.py#L209-L212

**Comment by fchollet** [comment]
> Is there any reason both scale and alpha parameters are omitted in selu activation here

They're not included in the backend ops either. This looks normal.

**Comment by Frightera** [comment]
@fchollet Thanks for the info. Can you review it once more?

**Pull request merged**